File: testData/action/after/CreateMethodFromUsage.scala
package com.test

import org.concordion.integration.junit4.ConcordionRunner
import org.junit.runner.RunWith

@RunWith(classOf[ConcordionRunner])
class CreateMethodFromUsage {

  def getField: A = null

  def method: A = null

  class A {
  }

  def createMe(arg: String, field: A, method: A, param3: String, param4: String, param5: String): String = ???
}

File: testData/action/after/CreateMethodFromUsage.java
package com.test;

import org.concordion.integration.junit4.ConcordionRunner;
import org.junit.runner.RunWith;

@RunWith(ConcordionRunner.class)
public class CreateMethodFromUsage {

    public A field;

    public A method() {
        return null;
    }

    public String createMe(String arg, A field, A method, String param3, String param4, String param5) {
        return null;
    }

    public static final class A {

    }
}
